packages/opentelemetry-exporter-jaeger/src/transform.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Link, CanonicalCode } from '@opentelemetry/types';
+import { ReadableSpan } from '@opentelemetry/basic-tracer';
+import {
+  ThriftSpan,
+  Tag,
+  Log,
+  ThriftTag,
+  ThriftLog,
+  ThriftUtils,
+  Utils,
+  ThriftReference,
+  TagValue,
+} from './types';
+
+const MICROS_PER_MILLI = 1000;
+
+/**
+ * Translate OpenTelemetry ReadableSpan to Jaeger Thrift Span
+ * @param span Span to be translated
+ */
+export function spanToThrift(span: ReadableSpan): ThriftSpan {
+  const traceIdHigh = span.spanContext.traceId.slice(0, 16);
+  const traceIdLow = span.spanContext.traceId.slice(16);
+  const parentSpan = span.parentSpanId
+    ? Utils.encodeInt64(span.parentSpanId)
+    : ThriftUtils.emptyBuffer;
+
+  const tags = Object.keys(span.attributes).map(
+    (name): Tag => ({ key: name, value: toTagValue(span.attributes[name]) })
+  );
+  tags.push({ key: 'status.code', value: span.status.code });
```

Comment:
This will appear as just an int as far I can tell, maybe we should add some sort of `code.toString()`?

Reply:
The `TagValue` can be either typeof `string` or `number` or  `boolean`.

Reply:
Oh I meant that the actual value for, say, `Canonical.UNKNOWN` is 2, which is won't mean anything when viewing the span. The value that goes into the tag should probably be `unknown`, `ok`, etc.

Reply:
Oh I see, `CanonicalCode[span.status.code]` would work then. To me status code always represented as a number. But I am open to make change, let me know.

Reply:
I guess the thing to me that makes it a little confusing (from a user pov) is that, unless I have experience with the grpc status codes, it's not the same as http status codes which most are familiar with. How about keeping the status.code and adding `tags.push({ key: 'status.name': value: CanonicalCode[span.status.code]})`?

Reply:
sure, done in https://github.com/open-telemetry/opentelemetry-js/pull/231/commits/c3a787afce9b718aaf0db503be951b3e984f7343